public/pages/Destinations/containers/CreateDestination/EmailRecipients/utils/helpers.js

```diff
@@ -15,13 +15,13 @@
 
 export default async function getEmailGroups(httpClient, searchText = '') {
   try {
-    const response = await httpClient.get(
-      `../api/alerting/destinations/email_groups?search=${searchText}&size=200`
-    );
-    if (response.data.ok) {
-      return response.data.emailGroups;
+    const response = await httpClient.get('../api/alerting/destinations/email_groups', {
+      query: { search: searchText, size: 200 },
```

Comment:
unrelated to these changes - rather than hardcoding a limit of 200 here, declare as a constant somewhere instead? Seems there's a few places where limits like this are hardcoded

Reply:
Thanks. Make sense. I think I can change it based on your idea in another PR.